docs/src/tari_script_no_op_vulnerability.md

```diff
@@ -0,0 +1,82 @@
+# TariScript No-Op script vulnerability
+
+This document describes a vulnerability in the current version of TariScript found by [@mikethetike](https://github.com/mikethetike).
+
+The initial specification of how TariScript is incorporated into a Tari Mimblewimble (MW) style transaction is defined in the Tari [RFC-0201](https://rfc.tari.com/RFC-0201_TariScript.html). 
+
+This vulnerability applies to transactions where a Sender is spending one of their inputs that is bound by a NO_OP script like is suggested in [RFC-0201](https://rfc.tari.com/RFC-0201_TariScript.html#standard-mw-transaction)
```

Comment:
```suggestion
This vulnerability applies to transactions where a Sender is spending one or more their inputs where all the inputs are bound by a NO_OP script like is suggested in [RFC-0201](https://rfc.tari.com/RFC-0201_TariScript.html#standard-mw-transaction)
```

Reply:
This is only a problem where all the inputs and all the outputs are changeable. 

if 
\\( \gamma = k_{sa1} + k_{sa2} - k_{Ob}U_b  \\)
Then Bob cant construct a k_{Ob} to satisfy (k_{sa1} + k_{sa2})

Reply:
Sure updated the wording